tools/update-dll-msvc.bat

```diff
@@ -0,0 +1,57 @@
+@echo off
+rem Update the DLL using MSVC.
+rem If the old DLL is in use, rename it to avoid compilation error.
+rem
+rem usage: update-dll-msvc [arch] [makeopts]
+rem
+rem   [arch] is 32 or 64. If omitted, it is automatically detected from the
+rem   %PROCESSOR_ARCHITECTURE% environment.
+rem   [makeopts] is option(s) for nmake.
```

Comment:
Are these comments correct?
I don't think that [arch] and [makeopts] are used in this batch file.

Reply:
@k-takata , yes, you are right, the makeopts should not be used for nmake.